### ISSUE TITLE
[Bug] Teams Adaptive Card Show as Blank on Android Mobile Client [Critical]

### ISSUE THREAD

**Issue by johnmiroki** [MicrosoftDocs/msteams-docs#11050]
### Steps to reproduce

Install the latest Teams
Install Developer Portal from Teams app store
Send the simplest card:

{
    "type": "AdaptiveCard",
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
    "version": "1.5",
    "body": [
        {
            "type": "TextBlock",
            "text": "New TextBlock",
            "wrap": true
        }
    ]
}

### Expected behavior

An adaptive card with some text shows in the chat with Developer Portal

### Actual behavior

A blank message is received (ignore the fact that screenshot is not taken from Developer Portal but the idea is the same)
![CleanShot 2024-06-13 at 18 37 21@2x](https://github.com/MicrosoftDocs/msteams-docs/assets/1551376/cbdd0ac4-7e2d-4183-92a6-49c6b3cc28d3)

**Comment by johnmiroki** [comment]
Teams client info
![image](https://github.com/MicrosoftDocs/msteams-docs/assets/1551376/6c1ebd59-431f-45f0-97af-4da9457339d0)

**Comment by Prasad-MSFT** [comment]
Thanks for raising your issue. We will check this at our end and will update you accordingly.

**Comment by Prasad-MSFT** [comment]
Hi @johnmiroki - We tried to repro this issue using the card JSON provided sending via Teams developer portal and observed that the card renders properly in both MS Teams Desktop/Web client and Android client.

**Android**:
![image](https://github.com/MicrosoftDocs/msteams-docs/assets/62237649/e0df9134-5591-4dde-9d1a-e623320c61b4)

**MS Teams Version**:
![image](https://github.com/MicrosoftDocs/msteams-docs/assets/62237649/64471296-2504-4f57-8eb4-1a6d455ffb6e)

**Comment by Prasad-MSFT** [comment]
Yes @johnmiroki, we tested with Microsoft Teams app installed in Android device.

**Comment by johnmiroki** [comment]
We tested on Teams downloaded from Chinese app store and we can reproduce the problem (limited tests on limited devices). 
Is Teams client repackaged for China market by some Microsoft partner?

**Comment by johnmiroki** [comment]
Now we think the issue only exists in the so called "21Vianet" version, any chance we can get a hold of their team?
![image](https://github.com/MicrosoftDocs/msteams-docs/assets/1551376/08e52d64-c465-48d0-bd43-a2816229ccb6)

**Comment by Prasad-MSFT** [comment]
We have raised a bug for the same. We will inform you once we get any further update from engineering team. 

Thanks!

**Comment by Prasad-MSFT** [comment]
Hi @johnmiroki, several fixes have been made with latest Teams upgrade. Could you please let us know if you are still encountering the issue?
